other/containers/tilck-ci-build/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:20.04
+# Also tried to use debian/stable-slim as base, but image size went up compared
+# to ubuntu 20.04
+ENV DEBIAN_FRONTEND=noninteractive
+# We don't any interactive features of apt, so we just disable them.
+RUN apt update && apt upgrade -y && \
+apt install git-core gcc g++ cmake imagemagick clang gcc-multilib g++-multilib \
+python3-minimal apt gawk unzip parted wget qemu-system-x86 -y && \
+mkdir -p tmp-tilck && \
```

Comment:
Essential step: You need to first create /tc/toolchain2, then clone tilck, then make tilck/toolchain2 to be a symlink pointing to /tc/toolchain2. Currently, the `syscc` target breaks.

Reply:
Solved exactly in this way. Now all tests pass.